src/msgwindow.h

```diff
@@ -53,11 +53,14 @@ typedef enum
 
 
 void msgwin_status_add(const gchar *format, ...) G_GNUC_PRINTF (1, 2);
+void msgwin_status_add_string(const gchar *msg);
 
 void msgwin_compiler_add(gint msg_color, const gchar *format, ...) G_GNUC_PRINTF (2, 3);
+void msgwin_compiler_add_string(gint msg_color, const gchar *msg);
```

Comment:
you should remove the duplicated prototype from the `GEANY_PRIVATE` section below